src/components/Domains.jsx

```
@@ -1,190 +1,136 @@
-              style={{ fontSize: 'clamp(3rem, 6vw, 6rem)' }}
+      <div className="max-w-7xl mx-auto">
+        
+        {/* --- HEADER --- */}
+        <div className="mb-16 border-b border-[#333] pb-8 flex flex-col md:flex-row justify-between items-end gap-6">
+            <div>
+                <motion.span 
+                    initial={{ opacity: 0, x: -20 }}
+                    whileInView={{ opacity: 1, x: 0 }}
+                    className="text-[#ccff00] text-sm font-bold tracking-widest uppercase mb-2 block"
+                >
+                    // System Capabilities
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)